posthog/migrations/0131_add_plugins_updated_created_at.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.11 on 2021-03-09 13:19
+
+import datetime
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0130_dashboard_creation_mode"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="plugin",
+            name="created_at",
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2020, 1, 1, 0, 0)),
+            preserve_default=False,
+        ),
+        migrations.AddField(model_name="plugin", name="updated_at", field=models.DateTimeField(auto_now=True),),
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="created_at",
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2020, 1, 1, 0, 0)),
```

Comment:
These defaults look weirdly hardcoded, shouldn't they be `django.utils.timezone.now`?

Reply:
Good question. I left a weird default so we're able to distinguish "old" plugins vs new ones after this PR. Having .now() would be misleading whenever looking at the db.